<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    background-color:rgb(0, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PTI picture SPCK/BOOK.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BacktoU1_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>720</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightgreen;</string>
    </property>
    <property name="text">
     <string>Back to unit1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_next5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>720</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightgreen;</string>
    </property>
    <property name="text">
     <string>Next lesson</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: violet;</string>
    </property>
    <property name="text">
     <string>Bag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: violet;</string>
    </property>
    <property name="text">
     <string>Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: violet;</string>
    </property>
    <property name="text">
     <string>Pencil</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PTI picture SPCK/teacher.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: pink;</string>
    </property>
    <property name="text">
     <string>Table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>290</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: pink;</string>
    </property>
    <property name="text">
     <string>Teacher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>290</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: pink;</string>
    </property>
    <property name="text">
     <string>Bag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>630</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: violet;</string>
    </property>
    <property name="text">
     <string>Chair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>630</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: violet;</string>
    </property>
    <property name="text">
     <string>Book</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PTI picture SPCK/chair.jfif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="right2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: violet;</string>
    </property>
    <property name="text">
     <string>Table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong7">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>630</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: pink;</string>
    </property>
    <property name="text">
     <string>Teacher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>630</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: pink;</string>
    </property>
    <property name="text">
     <string>Pencil</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong8">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>630</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: pink;</string>
    </property>
    <property name="text">
     <string>Table</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PTI picture SPCK/pencil.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
